src/runtime/pkg/resourcecontrol/utils.go

```diff
@@ -36,9 +36,10 @@ func DeviceToCgroupDeviceRule(device string) (*devices.Rule, error) {
 	case unix.S_IFCHR:
 		deviceRule.Type = 'c'
 	case unix.S_IFBLK:
+	case unix.S_IFREG:
 		deviceRule.Type = 'b'
```

Comment:
In Go, case statements do NOT fall through, meaning this code is now equivalent to below - is this intended?
```golang
	case unix.S_IFBLK:
        // Do nothing, deviceRule.type is NOT set
	case unix.S_IFREG:
		deviceRule.Type = 'b'
```